tests/integ/test_byo_estimator.py

```diff
@@ -58,10 +57,9 @@ def test_byo_estimator(sagemaker_session, region, cpu_instance_type):
 
     with timeout(minutes=TRAINING_DEFAULT_TIMEOUT_MINUTES):
         data_path = os.path.join(DATA_DIR, "one_p_mnist", "mnist.pkl.gz")
-        pickle_args = {} if sys.version_info.major == 2 else {"encoding": "latin1"}
 
         with gzip.open(data_path, "rb") as f:
-            train_set, _, _ = pickle.load(f, **pickle_args)
+            train_set, _, _ = pickle.load(f, encoding="latin1")
```

Comment:
same batch of code multiple places. make helper function?

Reply:
-sigh- you're not wrong...

Reply:
I didn't remove as much repetition as I probably could've because I wasn't convinced that this warranted a top-level fixture, but I did at least create a helper method so that the `pickle.load` call appears only once